proto/injective/peggy/v1/msgs.proto

```
@@ -427,6 +434,10 @@ message MsgUpdateRateLimit {
 
   // new_rate_limit_window is the new length of the sliding window
   uint64 new_rate_limit_window = 5;
+
+  // new_token_oracle_type is used to obtain the USD price of the ERC20 token
+  // (in relation with token_price_id)
+  injective.oracle.v1beta1.OracleType new_token_oracle_type = 6;
```

Buf (1.71.0)

[error] cannot find `injective.oracle.v1beta1.OracleType` in this scope

(COMPILE)